vllm/model_executor/models/minicpmv.py

```diff
@@ -407,10 +1371,14 @@ def init_llm(self, config, cache_config, quant_config):
             return MiniCPMForCausalLM(config,
                                       cache_config=cache_config,
                                       quant_config=quant_config)
-        else:
+        elif self.version == 2.5:
             return LlamaForCausalLM(config,
                                     cache_config=cache_config,
                                     quant_config=quant_config)
+        else:
+            return Qwen2ForCausalLM(config,
+                                    cache_config=cache_config,
+                                    quant_config=quant_config)
 
     def init_vision_module(self):
         if self.version == 2.0:
```

Comment:
Can we detect the model version without an explicit version number? (as per my comment in #6814)